### PR TITLE
docs(byrole.mdx): add tips for input type password

### PR DESCRIPTION
**Pull request by moeyashi**
from https://github.com/testing-library/dom-testing-library/issues/1128

### CODE REVIEW

File: docs/queries/byrole.mdx [hunk 1]
```diff
@@ -73,6 +73,14 @@ not replace the functionality of these attributes. For example
 its description if `fancy.jpg` could not be loaded. Whether you want to assert
 this functionality in your test or not is up to you.
 
+:::tip input type password
+
+Use [ByLabelText](queries/bylabeltext.mdx) to select `<input type="password" />`.
```
Comment (MatanBobi) [inline docs/queries/byrole.mdx:78]: Thanks!
I think it's worth to first explain the reason and then come with alternatives.
Something like:
```
Unfortunately, the spec defines that `<input type="password" />` has no implicit role. This means that in order to query this type of element we must fallback to a less powerful query such as `[ByLabelText](queries/bylabeltext.mdx)`.
```
Wdyt?

Comment (moeyashi) [reply docs/queries/byrole.mdx:78]: Thank you for the review.

To conclude, I don't find either approach particularly jarring.

I considered whether to put the reasoning before or after the conclusion, but in Japan, it is customary to present the conclusion first, so I followed that practice. (However, I’m not entirely confident that this is the best approach for documentation.)
Additionally, when I searched within the repository using :::tip, I found that imperative statements were used here, which also influenced my decision.
https://github.com/testing-library/testing-library-docs/blob/main/docs/svelte-testing-library/api.mdx?plain=1#L113-L117

Comment (moeyashi) [reply docs/queries/byrole.mdx:78]: > ByLabelText is only one solution.

> it's not just a decision we took as part of the library

I 100% agree.

I also think that the expression you proposed effectively includes these points and is very appropriate.

Thank you for your explanation.

Comment (MatanBobi) [reply docs/queries/byrole.mdx]: Thank you for taking the time to work on this.

**Comment by MatanBobi** [comment]
@all-contributors please add @moeyashi for docs :)

**Pull request merged**